#52-loan-notifications

topless opened the pull request:
Implements #52 and resolves user story #24.

On after method of a transition we fire a signal with the class that was triggered and the loan that was affected.

You can try it like

```
@loan_state_changed.connect
def listen_loan_changes(sender, loan, **extra):
    print(sender, loan)
```

zzacharo commented on invenio_circulation/transitions/base.py:
```diff
@@ -115,4 +116,5 @@ def after(self, loan):
         """Commit record and index."""
         loan['transaction_date'] = loan['transaction_date'].isoformat()
         loan.commit()
+        loan_state_changed.send(self, loan=loan)
```
just to be typical put this line after the TODO because we should notify when the save and indexing has been completed successfully. Then we can merge it.

topless committed:
loan: notifications #52

Implements #52 and resolves user story #24.

On after method of a transition we fire a signal with the class that was triggered and the loan that was affected.
